.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
What the point of this empty file?

Reply:
Well, it contains `14` which is not something empty